docs/performance/tune-performance.md

```diff
@@ -126,8 +126,7 @@ import onnxruntime as rt
 
 so = rt.SessionOptions()
 so.graph_optimization_level = rt.GraphOptimizationLevel.ORT_ENABLE_ALL
-session = rt.InferenceSession(model, sess_options=so)
-session.set_providers(['CUDAExecutionProvider'])
+session = rt.InferenceSession(model, sess_options=so, providers=['CUDAExecutionProvider'])
```

Comment:
Starting 1.10, providers needs to be explicitly provided at session construction time